### DISCUSSION TITLE
Going from bus to multiple input gate in a nice way?

### DISCUSSION THREAD

**Discussion by mengstr** [hneemann/Digital#996] [Q&A] [answered]
This is annoying me a bit - I wish there was a way to make gates not have a "hole" in the middle of the inputs when the number is even.  Or in there another way of doing this in a nicer/better way?
<img width="726" alt="Screenshot 2022-07-26 at 10 48 47" src="https://user-images.githubusercontent.com/325326/180967386-b5176360-79cb-464d-b2c3-ec955382a2fa.png">

**Comment by hneemann** [comment, marked as answer]
A more elegant way - at least as far as the layout of the circuit is concerned - in this particular case would be to use a comparator. Otherwise, there is no better way.
